gecosws_config_assistant/controller/MainMenuController.py

```diff
@@ -295,6 +296,7 @@ def checkForUpdates(self):
             apt_progress.show_terminal(True)
             try:
                 cache.commit(apt_progress.acquire, apt_progress.install)
+                time.sleep(5)
```

Comment:
What is this arbitrary sleep?

Reply:
This instruction avoids going from one screen to another too quickly for the user to read the update information

Reply:
I am not sure about a fixed time sleep. If you are showing info that must be read, a confirm button is needed. If it is just something you can oversee, then this sleep is not needed at all.